src/traces/parcoords/parcoords.js

```diff
@@ -227,8 +234,6 @@ function styleExtentTexts(selection) {
     selection
         .classed('axisExtentText', true)
         .attr('text-anchor', 'middle')
-        .style('font-weight', 100)
```

Comment:
@monfera is removing that font-weight setting on purpose? Unsurprisingly, it makes the image test fail.